restmanager/fridges/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.0.3 on 2020-02-27 13:33
+# Generated by Django 3.0.3 on 2020-05-02 18:41
```

Comment:
this is generated by django should it be in git?

Reply:
Lisätty git ignoreen migrations kansion sisältö.